src/proto/protocol.pb.h

```diff
@@ -27,12 +27,13 @@ typedef struct _ledger_swap_NewFundResponse {
 /* @@protoc_insertion_point(struct:ledger_swap_NewFundResponse) */
 } ledger_swap_NewFundResponse;
 
+typedef PB_BYTES_ARRAY_T(33) ledger_swap_NewTransactionResponse_payin_extra_data_t;
 typedef PB_BYTES_ARRAY_T(16) ledger_swap_NewTransactionResponse_amount_to_provider_t;
 typedef PB_BYTES_ARRAY_T(16) ledger_swap_NewTransactionResponse_amount_to_wallet_t;
 typedef PB_BYTES_ARRAY_T(32) ledger_swap_NewTransactionResponse_device_transaction_id_ng_t;
 typedef struct _ledger_swap_NewTransactionResponse {
     char payin_address[63];
-    char payin_extra_id[33];
+    char payin_extra_id[20];
```

Comment:
Initial plan was to use this field to convey the hash
I've already made and merged a PR when the spec was finalized
Upon implementation we can't use `payin_extra_id` because it's char array and not byte array and we're worried of passing '\0' bytes in the string
So it was decided to introduce a new field `payin_extra_data` of 33 bytes
This PR reverts `payin_extra_id` to 20 as it was before